### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by lfundaro**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,3 +8,4 @@ hey jo !
 
 
 dd
+12
```
Comment (lfundarotest) [inline README.md:11]: this is great

**Review by lfundarotest — COMMENTED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (lfundaro) [reply README.md:11]: are you sure about this ?

Comment (lfundarotest) [reply README.md:11]: I see what you mean !

Comment (lfundaro) [reply README.md:11]: > I see what you mean !

impossible

Comment (lfundaro) [reply README.md:11]: what about we do something else ?

Comment (lfundarotest) [inline README.md:11]: we need to reconsider

Comment (lfundaro) [reply README.md:11]: are you serious ?

**Pull request closed without merging**